Marlin/pins_5DPRINT.h

```diff
@@ -46,29 +46,28 @@
 //
 // Steppers
 //
+// Microstepping pins - Mapping not from fastio.h (?)
 #define X_STEP_PIN          0
 #define X_DIR_PIN           1
 #define X_ENABLE_PIN       23
+#define X_MS1_PIN          25
+#define X_MS2_PIN          26
```

Comment:
By the way, when contributor update the commented point, review should be closed automatically by GitHub...
From my side (Win10 + Chrome), this review is still opened for some reason.